packages/ws-worker/test/events/run-start.test.ts

```diff
@@ -91,12 +91,18 @@ test('run:start should log the version number', async (t) => {
   const input = 'abc';
   const jobId = 'job-1';
 
+  // explicitly load package.json her
+  // because load-version does it a bit differently
+  const { default: pkg } = await import('../../package.json', {
+    with: { type: 'json' },
+  });
```

Comment:
this seem to only work for 20+

import with assertion doesn't seem to be stable at the moment.

Reply:
Yes, this is fine - I'm only running the unit tests in node 22. Runtime code will be absolutely fine.

I've chosen import over readFile because it doe a better job of resolving the module relatively. `fs` was returning the wrong package.json.

Saying that, I've just noticed that everything is failing...

Reply:
I think the aim wasn't to only make the tests pass but also be sure that when someone with other node supported version runs the code it just works.

Reply:
Not for this repo, no. Devs should be using asdf to manage versions and running on node 22.

The npm packages should all be flexible, but the repo itself should not